### ISSUE TITLE
After install, cannot log in to /admin

### ISSUE THREAD

**Issue by jeffstoner** [Webiny/Webiny#4]
Following directions here: https://www.webiny.com/hub/tutorials/installing-webiny-on-vagrant
Host system: Fedora 26 x64 (fully patched)
Vagrant: 2.0.0 installed using Hashicorp rpm
VirtualBox 5.2 installed from Oracle repo
I patched Vagrant to use VirtualBox 5.2, according to https://github.com/hashicorp/vagrant/issues/9090#issuecomment-338084000

1. During "vagrant up", I found that /home/vagrant/code in the guest is not NFS mounted. Unknown cause (I have disable firewalld and set SELinux to permissive mode.)
2. I ignored that and ran "yarn add webiny-cli" only to fail because the ownership of /home/vagrant/code was root:root, which I had to change to vagrant:vagrant to get webiny-cli to install.
3. After running "webiny-cli" and answering the questions, I tried loading the site but it failed, displaying the PHPInfo() page. Although the setup did _not_ show any message regarding nginx, I found that it did not copy the vhost.conf to /etc/nginx/sites-enabled/ directory. I copied the file to /etc/nginx/sites-available/ and symlinked it to /etc/nginx/sites-enabled and restarted nginx.
4. I I tried loading the site's /admin resource but got a Smarty exception about being unable to create a directory. I looked in the guest and found that the /home/vagrant/code/Cache folder was missing completely. I created it and 'chgrp www-data /home/vagrant/code/Cache' and was able to load the /admin page
5. I could not log in with the user/pass I specified during setup. I keep getting the message "TypeError: Cannot read property 'twoFactorAuth' of null" It is at this point that I am completely blocked from trying anything with Webiny.
Any help resolving this, clue-by-4 upside the head, or drop-kicks to the brain would be appreciated.

Side note: I did notice two settings in /home/vagrant/code/Configs/Base/Security.yaml:
```yaml
Security:
    TwoFactorAuth:
        Key: removed
```
and 
```yaml
Login:
    SecurityFirewall: Webiny
    2FactorAuth: false
```
I would suggest being consistent throughout all code/configs by using either "TwoFactorAuth" or "2FactorAuth" - not both.

**Comment by SvenAlHamad** [comment]
Hey @jeffstoner 

just to let you know that we are working on creating an install application that will fully automatize Webiny setup on your local machine, and this error should also go away.

We are finalizing the installer for Windows and MacOs and then will work on the unix one. Should take another week or two.

Once we are ready, I'll give you a ping via this ticket, and then you can let us know if the issue is resolved.

As for the 2FA side note, point taken :) I'll see we clean that up in one of the next releases.

Cheers

**Comment by jeffstoner** [comment]
/me destroys vagrant vm, makes sandwich, and waits

**Comment by SvenAlHamad** [comment]
Hey @jeffstoner,

appreciate your patience...it took us bit more time than originally planned to get the installers up and running, but it's done now 🎉 

You can download the installer from this page for your OS: https://www.webiny.com/download

Since you've tried installing manually the first time, I suggest you remove all the files and destroy the vagrant machine, this way the installer will pull the newest vagrant box.

Let me know how it goes :)

**Comment by jeffstoner** [comment]
Installer downloaded and installed. I ran it...and got an error. The issue was...<drumroll/> Vagrant. My recent upgrade to Fedora 27 replaced my Vagrant rpm from Hashicorp with the (older) version in Fedora's repository (which kinda pissed me off.) So, I ripped it out and downloaded 2.0.1 from Hashicorp and installed it (incidentally, this fixed the issue with using Virtualbox 5.2, too.)
Running the installer and choosing the Vagrant build worked. I spun up a box and watched it pull in the Webiny v0.1.6 image. So, things are looking much better!
I'll give the Docker build a whirl while I'm at it.

Thank you!
